### PR TITLE
implement Nutanix CSI Driver object

### PR DESCRIPTION
**Pull request by tuxtof**
Implement New driver name for Nutanix CSI storage

### CODE REVIEW

File: operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml-patch [hunk 1]
```diff
@@ -17,3 +17,4 @@
       - csi.sharedresource.openshift.io
       - diskplugin.csi.alibabacloud.com
       - vpc.block.csi.ibm.io
+      - csi.nutanix.com
```
Comment (JoelSpeed) [inline operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml-patch:20]: Nit, there should be a new line after this

File: operator/v1/types_csi_cluster_driver.go [hunk 2]
```diff
@@ -55,6 +55,7 @@ const (
 	SharedResourcesCSIDriver CSIDriverName = "csi.sharedresource.openshift.io"
 	AlibabaDiskCSIDriver     CSIDriverName = "diskplugin.csi.alibabacloud.com"
 	IBMVPCBlockCSIDriver     CSIDriverName = "vpc.block.csi.ibm.io"
+	NutanixCSIDriver         CSIDriverName = "csi.nutanix.com"
```
Comment (JoelSpeed) [inline operator/v1/types_csi_cluster_driver.go:58]: Are there different types of storage on Nutanix or just the one?

Comment (tuxtof) [reply operator/v1/types_csi_cluster_driver.go:58]: Even if différent mode only one CSI and all is managed in storage class

File: operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml-patch (hunk 1 above)
Comment (tuxtof) [reply operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml-patch:20]: Ok i will add it
Thanks @JoelSpeed

**Comment by gnufied** [comment]
cc @jsafrane @bertinatto

**Comment by jsafrane** [comment]
The PR is on hold because the CSI driver is not (yet?) open source and we don't have a way how to ship proprietary binary blobs as part of OpenShift payload.
The code itself looks good.

File: operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml-patch (hunk 1 above)
Comment (tuxtof) [reply operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml-patch]: fixed in [18b340e](https://github.com/openshift/api/pull/1138/commits/18b340ed1d5f051eb97540ec93049dbbc6a26395)

Comment (bertinatto) [inline operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml-patch:20]: Please add this to the `operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml` file as well

Comment (tuxtof) [reply operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml-patch:20]: ok @bertinatto i just add it to in the file with [7ee0d1f](https://github.com/openshift/api/pull/1138/commits/7ee0d1fe8892b8825df5353ca342842d490fe587)
but it was not specified in the doc you share with us , i imagined it was automatically generated base on the patch file

**Pull request closed without merging**